openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -52,6 +52,15 @@ def add_credit_course(self, enabled=True):
 
         return credit_course
 
+    def add_credit_eligible_course(self, credit_course, username):
```

Comment:
Rename. The current name implies that you care creating a course. `mark_user_credit_eligible` would be better.